tools/memap.py

```diff
@@ -451,6 +454,8 @@ def generate_output(self, export_format, file_output=None):
 
         return True
 
+    toolchains = ["ARM", "ARM_STD", "ARM_MICRO", "GCC_ARM", "IAR"]
```

Comment:
I agree. I think that any changes to the logic that is not directly related to the argument parsing should be in another PR. Let's try to not make this PR any bigger than it already is :stuck_out_tongue_winking_eye: